src/app/core/components/header/header.component.html

```diff
@@ -5,6 +5,7 @@
     <div class="govuk-grid-row govuk-!-margin-bottom-2 govuk-width-container">
       <div class="govuk-grid-column-one-half">
         <a href="/" id="header-home-link" class="govuk-header__link govuk-header__link--homepage">Vehicle testing management</a>
+        <span id="appversion">{{  getAppVersion() }}</span>
```

Comment:
This needs a `v` in front of it, e.g. `<span id="appversion">v{{ version }}</span>`